BinOrj/SalarySystem
# Develop (#11)

* Test pull request (#4)

* Changes in workflow

* Update dotnet.yml

* More changes to main

* Update dotnet.yml

* Broken test for scientific research

* Fixed test, resolves #3

* Team Test

Co-authored-by: Emil Örjes <74154767+emilorjes@users.noreply.github.com>

* Test

* Playing with the workflow

* changes in workflow

* Added exclomationmark

* Update README.md

* Added map structure.

* Added Controllers

* Added Employees and Roles class, iplemented Roles enum.

* Added Interfaces And Utility class

* Implemented IAccount

* Implemented Ilogger

* Implemented Employees and modified IAccount

* Added demo testcase

* Testing login

* Changes in unit test

* More testing

* Crazy commit

* Added menu handling

* Added ListRoles method and delete check of account.

* Changed name from UserMenu to EmployeeMenu

* Update dotnet.yml

* Update dotnet.yml

* Update dotnet.yml

* Update dotnet.yml

* Update dotnet.yml

* Update dotnet.yml

* Update dotnet.yml

* Update dotnet.yml

* Update dotnet.yml

Co-authored-by: Emil Örjes <74154767+emilorjes@users.noreply.github.com>
Co-authored-by: Emil Örjes <emil.orjes@hotmail.com>
